tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -61,11 +61,13 @@ Docker logs with follow and tail
     Should Be Equal As Integers  ${rc}  0
     # Wait for the first 10 lines to be logged
     Wait Until Keyword Succeeds  20x  200 milliseconds  Grep Logs And Count Lines  ${id}  5  10
+    # --tail=5 to skip the first 5 lines and --follow to wait for the rest
+    ${proc}=  Start Process  docker ${params} logs --tail 5 --follow ${id}  shell=True
```

Comment:
could still have a race here if the logs process starts tailing after the next step in the test is run.
Is there any way to check for non-nil output so we can validate this command has actually taken effect before the kill is dispatched?